docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Use an official PHP-FPM base image
+FROM php:8.4-fpm
```

Comment:
Similar comment about using ARG to get the tag for php base-image. It's a laziness habit I've had for a while

Reply:
I kinda want to keep this one up for the user to change, as I've fixed the php version for the project to be 8.3 and up.

Need to push others to use new versions 😆

Reply:
The build arg is for you, so that if someone needs to, they PR it, rather than you needing to, but I'll retire the comment. 👍

Reply:
I don't see it as a blocker for anyone to send a PR.